content/docs/support-and-ops/ops-recipes/troubleshooting-cert-manager.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Cert manager Troubleshooting"
+owner:
+- https://github.com/orgs/giantswarm/teams/team-bigmac
+classification: public
+---
+
+# Table of Contents
+1. [DNS01 challenge issues](#dns01)
+
+## DNS01 challenge issues
+
+When there is a certificate not becoming ready, it is often due to the DNS/HTTP challenge not being completed. Check the cert manager controller logs:
+
+```bash
+stern cert-manager -s 1m -n kube-system -i CERT_NAME
+```
+
+In case you see errors like:
+
+```
+E0828 14:04:53.727973       1 sync.go:190] "cert-manager/challenges: propagation check failed" err="NS ns-1536.awsdns-00.co.uk.:53 returned REFUSED for _acme-challenge.my-app.gig3-prd.aws.cps.xxx.com." resource_name="my-app-cert-h8qxc-3214626457-1657644764" resource_namespace="aaa-prod-application" resource_kind="Challenge" resource_version="v1" dnsName="my-app.gig3-prd.aws.cps.xxx.com" type="DNS-01"
+```
+
+This is related to the DNS challenge not being successful. We found thanks to this article](https://community.letsencrypt.org/t/error-renewing-certificate-from-le-ns-returned-refused-for-acme-challenge/174132/1) when there was incident that point us to the right direction. The problem is due to cert manager base image which might use musl libc library which has a bug in it (ipv4/ipv6 resolution issue).
```

Comment:
IMO cert-manager uses distroless debian https://github.com/cert-manager/cert-manager/blob/master/hack/latest-base-images.sh#L29

Reply:
yeah you are right, it is a problem but not related to musl